cli-processor/src/main/java/gov/nist/secauto/metaschema/cli/processor/ExitCode.java

```diff
@@ -29,12 +29,34 @@
 import edu.umd.cs.findbugs.annotations.NonNull;
 
 public enum ExitCode {
+  /**
+   * The command executed without issue.
+   */
   OK(0),
+  /**
+   * The command executed properly, but the operation failed.
+   */
   FAIL(1),
+  /**
+   * The argument information for a command is invalid.
+   */
   INPUT_ERROR(2),
```

Comment:
MInor, non-blocking: since below this is used for errors characterized with messages like, `""The provided destination '%s' already exists and the '%s' option was not provided."`, perhaps it could be a little more general and be classified as an IO error? It seems it is not always input, but also output, so up to you.

```suggestion
  /**
   * The argument information for a command is invalid.
   */
  IO_ERROR(2),
```